webui/src/pages/repositories/repository/pulls/pullDetails.jsx

```diff
@@ -31,17 +32,54 @@ const StatusBadge = ({status}) => {
         case PullStatus.open:
             return <Badge pill bg={"success"}>{<GitPullRequestIcon/>} {text}</Badge>;
         case PullStatus.closed:
-            return <Badge pill bg={"purple"}>{<GitPullRequestClosedIcon/>} {text}</Badge>;
+            return <Badge pill bg={"secondary"}>{<GitPullRequestClosedIcon/>} {text}</Badge>;
         case PullStatus.merged:
-            return <Badge pill bg={"danger"}>{<GitMergeIcon/>} {text}</Badge>;
+            return <Badge pill bg={"primary"}>{<GitMergeIcon/>} {text}</Badge>;
         default:
-            return <Badge pill bg={"secondary"}>{text}</Badge>;
+            return <Badge pill bg={"light"}>{text}</Badge>;
     }
 };
 
 const PullDetailsContent = ({repo, pull}) => {
+    let [loading, setLoading] = useState(false);
+    let [error, setError] = useState(null);
+
+    const mergePullRequest = async () => {
+        // TODO: this is not ideal - this should be handled in an atomic way
```

Comment:
Let's open an issue for this and remove the comment as this is probably not something we're going to handle in the near future

Reply:
[Issue](https://github.com/treeverse/lakeFS/issues/8210).